csharp/HOLMS.Platform/HOLMS.Platform/Support/Time/InclusiveDateRange.cs

```diff
@@ -32,6 +32,8 @@ public abstract class InclusiveDateRange : IEnumerable<LocalDate> {
             End = other.End;
         }
 
+        public bool Overlaps(InclusiveDateRange other) => !(other.End < Start && End < other.Start);
```

Comment:
Thinking this through in my head, I don't think this is right. The Gedanken-Experiment:

*Postulate:* `other` is a date range that terminates before `this`
*Expectation:* `this.Overlaps == false`
*Observation 1:* `a = other.End < this.Start; a == true` 
*Observation 2:* `b = this.End < other.Start; b == false`
*Observation 3:* `Overlaps = !(a && b); Overlaps == true`
*Conclusion:* Overlaps should evaluate to false, but in this case evaluates to true. Therefor, the theorem that this code represents does not correctly evaluate the overlapping quality of two date ranges.